vllm/config.py

```diff
@@ -425,6 +425,9 @@ class ModelConfig:
     - "transformers" will use the Transformers model implementation."""
     override_attention_dtype: Optional[str] = None
     """Override dtype for attention"""
+    process_hidden_states: Optional[bool] = False
```

Comment:
If this can never be `None`, we shouldn't hint it as `Optional`
```suggestion
    process_hidden_states: bool = False
```